src/LayerEditor/ui/data/le_data.py

```diff
@@ -0,0 +1,296 @@
+"""Analyzis Editor static parameters
+
+.. codeauthor:: Pavel Richter <pavel.richter@tul.cz>
```

Comment:
Replace Pavel Richter with Tomas Blazek ! Just kidding. Remove author notes at all, author is known from the repository.